chainreg/chainregistry.go

```diff
@@ -5,7 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io/ioutil"
```

Comment:
nit: The preferred prefix for multi-package changing commits is `multi: ` instead of `misc: `.

Also, can you please update the commit message to mention the full function that's replaced? E.g. `multi: replace ioutil.ReadAll`? Thanks.